deckhouse-controller/pkg/controller/deckhouse-release/updater/updater.go

```diff
@@ -57,17 +60,17 @@ func (w *webhookDataGetter) GetMessage(release *v1alpha1.DeckhouseRelease, relea
 	return fmt.Sprintf("New Deckhouse Release %s is available. Release will be applied at: %s", version, releaseApplyTime.Format(time.RFC850))
 }
 
-func newKubeAPI(client client.Client, dc dependency.Container, imagesRegistry string) *kubeAPI {
-	return &kubeAPI{client: client, dc: dc, imagesRegistry: imagesRegistry}
+func NewKubeAPI(client client.Client, dc dependency.Container, imagesRegistry string) *KubeAPI {
+	return &KubeAPI{client: client, dc: dc, imagesRegistry: imagesRegistry}
 }
 
-type kubeAPI struct {
+type KubeAPI struct {
```

Comment:
Sorry, I'm a bit lost, but for what purpose are we making the structure and the method of creating the structure public?

Reply:
it's reused [here](https://github.com/deckhouse/deckhouse/pull/9081/files#diff-f5f1549dccbd7351a2389d6bc059b5f448e7e169efc48647e51bb86c2a1ec718R596)